frontend/playground-frontend/src/features/Train/features/Tabular/components/TabularParametersStep.tsx

```diff
@@ -75,8 +75,10 @@ const TabularParametersStep = ({
     : { data: undefined };
   const {
     handleSubmit,
-    formState: { errors, isDirty },
+    formState: { errors, isDirty, dirtyFields },
```

Comment:
dirtyFields doesn't seem to be used anywhere

Reply:
@anikethswain

Reply:
Good catch. Should be removed now and ready for merging